### PR TITLE
add database

### PR DESCRIPTION
**Pull request by miguelriverasarmiento**
this is the setup database, and is running state..
Captura de pantalla de 2021-02-15 12-37-37

### CODE REVIEW

File: — [hunk 1]
```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (jjmedinas) [inline —]: What is this file for??

**Review by jjmedinas — CHANGES_REQUESTED** [review]
What is this file for? `src/IMAG0492_2.jpg ` Can't you just remove it?
Same for my other comment.

File: — (hunk 1 above)
Comment (miguelriverasarmiento) [reply —]: Yes, I deleted the file src/IMAG because it was not necessary, and the file PGP  I think it's mongo file

Comment (jjmedinas) [reply —]: Hmm not sure, could you check if it still works after deleting it?
@miguelriverasarmiento

Comment (miguelriverasarmiento) [reply —]: I removed it and then it did not work and I put it back and it does not work either

**Review by jjmedinas — APPROVED** [review]

**Pull request merged**